api/src/LinkyLink/Controllers/LinksController.cs

```diff
@@ -0,0 +1,247 @@
+﻿using LinkyLink.Helpers;
+using LinkyLink.Models;
+using LinkyLink.Service;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Cosmos;
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Drawing.Printing;
+using System.IO;
+using System.Linq;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace LinkyLink.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class LinksController : ControllerBase
```

Comment:
how about class-level documentation that describes this class and its purpose on a high level?

Reply:
ofc, same for other classes as well

Reply:
Added